Cargo.toml

```diff
@@ -27,13 +27,13 @@ with-runtime = ["cairo-native-runtime"]
 
 [dependencies]
 bumpalo = "3.14"
-cairo-lang-compiler = "2.3.1"
-cairo-lang-defs = "2.3.1"
-cairo-lang-diagnostics = "2.3.1"
-cairo-lang-filesystem = "2.3.1"
-cairo-lang-lowering = "2.3.1"
-cairo-lang-sierra = "2.3.1"
-cairo-lang-sierra-generator = "2.3.1"
+cairo-lang-compiler = "2.4.0"
```

Comment:
is there a reason you are using 2.4.0 instead of 2.4.2?

Reply:
overlooked this dep, will update. The reason I was on `2.4.1` was because the contract I would like to test cairo-native against was compiled on 2.4.1, so didn't realize there was a `2.4.2` already. Will update!